### DISCUSSION TITLE
Create Tenant With existing Account

### DISCUSSION THREAD

**Discussion by meliora-solution** [JonPSmith/AuthPermissions.AspNetCore#118] [General]
I  used authp in our multitenant project.  So far so good.  I can create tenant for new user. Is there any scenario where user has already an account and then want create tenant.  How can I solve the this problem ?

**Comment by JonPSmith** [comment]
Hi @meliora-solution,

The the AuthP library has a few ways that you invite a new users and another ways where a new user can create a new tenant using an "Tenant User". Have at:

- Article ["Building ASP.NET Core and EF Core multi-tenant apps – Part2: Administration"](https://www.thereformedprogrammer.net/building-asp-net-core-and-ef-core-multi-tenant-apps-part2-administration/)
- Article ["Multi-tenant apps with different versions can increase your profits"](https://www.thereformedprogrammer.net/multi-tenant-apps-with-different-versions-can-increase-your-profits/)
- Plus look at the [AuthP Wiki](https://github.com/JonPSmith/AuthPermissions.AspNetCore/wiki), which has a LOT of information on how each part of the AuthP library
